README.md

```diff
@@ -514,11 +558,11 @@ To start the ballerina service using the configuration file, run the following c
 ```
    $ ballerina run restful_service/ --config restful_service/ballerina.conf
 ```
-NOTE: The above configuration is the minimum configuration needed to enable tracing and metrics. With these configurations default values are load as the other configuration parameters of metrics and tracing.
+>NOTE: The above configuration is the minimum configuration needed to enable tracing and metrics. With these configurations, default values are loaded for the rest of the configuration parameters of metrics and tracing.
```

Comment:
Minor corrections:
">NOTE: The configuration provided above is the minimum configuration required to enable tracing and metrics. When you use the minimum configuration, the default values are loaded for the rest of the configuration parameters of metrics and tracing."